skyportal/tests/frontend/test_user_management.py

```diff
@@ -62,6 +63,8 @@ def test_grant_and_delete_user_acl(driver, super_admin_user, user):
     filter_for_user(driver, user.username)
     driver.click_xpath(f'//*[@data-testid="addUserACLsButton{user.id}"]')
     driver.click_xpath('//*[@data-testid="addUserACLsSelect"]')
+    element = driver.find_element(By.XPATH, f'//li[text()="{acl}"]')
+    driver.execute_script("arguments[0].scrollIntoView();", element)
```

Comment:
same as the scroll to element and click, I believe we might even have a simple scroll to element. Please check the driver class and use that if available, if not then you can keep this of course, it's completely fine.